docs/data/base/components/portal/portal.md

```diff
@@ -8,18 +8,16 @@ packageName: '@mui/base'
 
 # Portal
 
-<p class="description">The portal component renders its children into a new "subtree" outside of the current DOM hierarchy.</p>
+<p class="description">The <code>Portal</code> component renders its children into a new subtree outside of the current DOM hierarchy.</p>
 
-The children of the portal component will be appended to the `container` specified.
-The component is used internally by the [`Modal`](/components/modal/) and [`Popper`](/components/popper/) components.
+The children of the `Portal` component will be appended to the specified container.
```

Comment:
Why would a developer need/want to use this component? I think it's worth explaining here.

Reply:
Maybe it's worth pointing out that our Portal is just a component version of [`React.createPortal`](https://reactjs.org/docs/portals.html) and was created for convenience. All the use cases of React's portal apply here as well.

Reply:
That is very helpful information @michaldudak! I'll draft up a new version to try to explain this.